src/nvme/mi.c

```diff
@@ -812,6 +865,38 @@ int nvme_mi_admin_identify_partial(nvme_mi_ctrl_t ctrl,
 	return 0;
 }
 
+int nvme_mi_service_primitive_control(nvme_mi_ep_t ep, __u8 opcode, __u16 *result_cpsr)
```

Comment:
Since this is new (public) API, we will need the cpsp value argument too, otherwise we will have to break compatibility this when introducing support for commands that pass a cpsp value.

Reply:
done